hubspot/plugin.py

```diff
@@ -0,0 +1,252 @@
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Dict, List, Sequence, Tuple
+
+import httpx
+
+from lutraai.augmented_request_client import AugmentedTransport
+
+
+@dataclass
+class HubSpotContactProperties:
+    first_name: str
+    last_name: str
+    email: str
+    hs_object_id: str
+    last_modified_date: datetime
+
+    # These are additional properties that are available in the HubSpot contact system
```

Comment:
Did you want this to appear as part of the doc_string so that code_gen has access to it?

Reply:
Yep, good point, thanks!